static/src/stylesheets/amp/_hosted.scss

```diff
@@ -334,6 +334,10 @@ $badge-width: $gs-column-width + $gs-gutter;
     bottom: 0;
     width: $badge-width;
     z-index: $zindex-ads;
+    margin: 0 $gs-gutter/2;
```

Comment:
Not that I am aware of, but if you find yourself using it a lot, you can make one :)